sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -522,7 +521,7 @@ public async Task UpdateAzureApplicationInsightsDataFeedWithMinimumSetupAndNewIn
 
             var dataFeedToUpdate = new DataFeed() { Description = description };
 
-            await adminClient.UpdateDataFeedAsync(disposableDataFeed.Id, dataFeedToUpdate);
+            await adminClient.UpdateDataFeedAsync(dataFeedToUpdate);
```

Comment:
same comment here about considering not doing the GET, or if you do the GET you can compare the `dataFeedToUpdate` with the `updatedDataFeed` and then with `dataFreedFromGet` or something like that

Reply:
Removed the extra GET from all DataFeed tests.